src/main/java/com/woowahan/techcourse/order/service/dto/request/CreateOrderRequestDto.java

```diff
@@ -0,0 +1,123 @@
+package com.woowahan.techcourse.order.service.dto.request;
+
+import com.woowahan.techcourse.order.domain.Order;
+import com.woowahan.techcourse.order.domain.OrderCoupon;
+import com.woowahan.techcourse.order.domain.OrderItem;
+import java.util.List;
+import java.util.stream.Collectors;
+import javax.validation.constraints.Min;
+import javax.validation.constraints.NotEmpty;
+import javax.validation.constraints.NotNull;
+import javax.validation.constraints.Positive;
+
+public class CreateOrderRequestDto {
+
+    @NotNull
+    @NotEmpty
+    private List<@NotNull CreateOrderCartItem> cartItems;
```

Comment:
```suggestion
    private List<@NotNull CreateOrderCartItem> cartItems;
```
이렇게도 보통 쓰나요? 처음보는 형태라서요
list에 대해 NotNull, NotEmpty를 했다면 충분하지 않을까요?

Reply:
아하 안쪽에 Null 이 들어왔을 경우에,
```json
{
cartItems:[null]
}

```
같이 들어오면 예외가 발생하지 않더라고요 그걸 막아보려고 했어요!